zebedee/zebedeeMapper/mapper.go

```diff
@@ -50,8 +51,20 @@ func MapZebedeeDatasetLandingPageToFrontendModel(dlp data.DatasetLandingPage, bc
 	sdlp.DatasetLandingPage.IsNationalStatistic = dlp.Description.NationalStatistic
 	sdlp.DatasetLandingPage.IsTimeseries = dlp.Timeseries
 	sdlp.ContactDetails = model.ContactDetails(dlp.Description.Contact)
-	sdlp.DatasetLandingPage.ReleaseDate = dlp.Description.ReleaseDate
-	sdlp.DatasetLandingPage.NextRelease = dlp.Description.NextRelease
+
+	// HACK FIX TODO REMOVE WHEN TIME IS SAVED CORRECTLY (GMT/UTC Issue)
+	if strings.Contains(dlp.Description.ReleaseDate, "T23:00:00"){
+		releaseDateInTimeFormat, err := time.Parse(time.RFC3339, dlp.Description.ReleaseDate)
+		if err != nil{
+			log.Error(err, nil)
+		}
+		sdlp.DatasetLandingPage.ReleaseDate = releaseDateInTimeFormat.Add(1 * time.Hour).Format("02 January 2006")
+
+	} else {
+		sdlp.DatasetLandingPage.ReleaseDate = dlp.Description.ReleaseDate
+	}
+	// END of hack fix	sdlp.DatasetLandingPage.NextRelease = dlp.Description.NextRelease
```

Comment:
Don't think `sdlp.DatasetLandingPage.NextRelease = dlp.Description.NextRelease` be commented out?